src/Aggregation/Composite.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Elastica\Aggregation;
+
+class Composite extends AbstractAggregation
+{
+    /**
+     * @return $this
+     */
+    public function setSize(int $size): self
+    {
+        return $this->setParam('size', $size);
+    }
+
+    /**
+     * @return $this
+     */
+    public function addSource(AbstractAggregation $aggregation): self
+    {
+        return $this->addParam('sources', [$aggregation]);
```

Comment:
Should we check that the aggregation is allowed, as a composite aggregation doesn't accept every type of aggregation?
ping @ruflin

Reply:
TBH I would leave this validation to the user. Reason is that in case other aggregations are added to the list, we need to update it each time.

Reply:
Yes let's resolve then ;)